plugins/services/src/js/stores/SDKEndpointStore.js

```diff
@@ -0,0 +1,149 @@
+import PluginSDK from "PluginSDK";
+
+import { SERVER_ACTION } from "#SRC/js/constants/ActionTypes";
+import AppDispatcher from "#SRC/js/events/AppDispatcher";
+import GetSetBaseStore from "#SRC/js/stores/GetSetBaseStore";
+
+import {
+  REQUEST_SDK_ENDPOINTS_SUCCESS,
+  REQUEST_SDK_ENDPOINTS_ERROR,
+  REQUEST_SDK_ENDPOINT_SUCCESS,
+  REQUEST_SDK_ENDPOINT_ERROR
+} from "../constants/ActionTypes";
+
+import SDKEndpointActions from "../events/SDKEndpointActions";
+import SDKServiceEndpoint from "../structs/SDKServiceEndpoint";
+
+class SDKEndpointStore extends GetSetBaseStore {
+  constructor() {
+    super(...arguments);
+
+    this.getSet_data = {
+      services: {}
+    };
+
+    PluginSDK.addStoreConfig({
+      store: this,
+      storeID: this.storeID,
+      listenAlways: true
+    });
+
+    this.dispatcherIndex = AppDispatcher.register(payload => {
+      if (payload.source !== SERVER_ACTION) {
+        return false;
+      }
+
+      const { type, data } = payload.action;
+      switch (type) {
+        case REQUEST_SDK_ENDPOINTS_SUCCESS:
+          this.processNewEndpoints(data.serviceId, data.endpoints);
+          break;
+        case REQUEST_SDK_ENDPOINTS_ERROR:
+          this.setService(data.serviceId, {
+            endpoints: [],
+            totalLoadingEndpointsCount: -1,
+            error: data.error.description
+          });
+          break;
+        case REQUEST_SDK_ENDPOINT_SUCCESS:
+          this.processNewEndpoint(
+            data.serviceId,
+            data.endpointName,
+            data.endpointData,
+            data.contentType
+          );
+          break;
+        case REQUEST_SDK_ENDPOINT_ERROR:
+          this.setService(data.serviceId, {
+            endpoints: [],
+            totalLoadingEndpointsCount: -1,
+            error: data.error
+          });
+          break;
+      }
+
+      return true;
+    });
+  }
+
+  getSDKEndpointService(serviceId) {
```

Comment:
Isn't the entire store about `SDK` endpoints? Why do we need the prefix?

Reply:
Removed